src/App.jsx

```diff
@@ -30,12 +30,12 @@ const HmdaHelp = makeAsyncComponent(() => import('./hmda-help'),
 const App = () => {
   const isFiling = !!window.location.pathname.match(/^\/filing/)
   const isHelp = !!window.location.pathname.match(/^\/hmda-help/)
-  const hideHeaderFooter = isFiling || isHelp
+  const hideCommonHeader = isFiling || isHelp
```

Comment:
maybe this can become `showCommonHeader`, then below, u can do `{showCommonHeader && <Rout ...` instead of having to do `null` later